### ISSUE TITLE
Create Future that resolves when invoice is marked as paid

### ISSUE THREAD

**Issue by D4nte** [comit-network/comit-rs#447]
For X->BTC-LN swap, as Alice, in `AlphaFunded` state, I need to have a Future that resolves once my LN invoice payment is done.
This will allow transition to `AlphaFundedBetaRedeemed` State.
`comit_node` manage such feature, polling LND on interval to check if specified invoice is paid.

DoD:
- [ ] Have an implementation of `LedgerEvents` for `LightningChannel` Ledger, Asset Bitcoin, struct AliceEvents.

Note: 
As soon as Beta is funded, Beta gets redeemed by Alice LND.
LND Invoice is Beta Identity (ie, as part of swap request).
LedgerEvents will have different implementations for Alice and Bob.
e2e test can be done in another ticket.
Dev will need to check how to fit LN in current traits.

Child of #431.
Blocks #457.
